tests/test_self.py

```diff
@@ -148,27 +159,32 @@ def test_all(self):
             [join(HERE, "functional", "a", "arguments.py")],
             reporter=MultiReporter(reporters),
             code=2,
+            bench=benchmark,
         )
 
-    def test_no_ext_file(self):
-        self._runtest([join(HERE, "input", "noext")], code=0)
+    def test_no_ext_file(self, benchmark):
+        self._runtest([join(HERE, "input", "noext")], code=0, bench=benchmark)
```

Comment:
Nitpick : We could name the variable in `_runtest` the same as the current one so it does `bench=bench` or `benchmark=benchmark` everywhere.